spec/models/exception_hunter/error_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+module ExceptionHunter
+  describe Error do
+    subject { build(:error) }
+
+    describe 'validations' do
+      it { is_expected.to validate_presence_of(:class_name) }
+    end
+
+    describe 'ocurred at' do
+      let!(:error) { create(:error, ocurred_at: nil) }
+
+      context 'when creating' do
+        it 'sets timestamp' do
+          expect(error.ocurred_at).not_to be_nil
```

Comment:
You could assert it sets the current time
```suggestion
          expect(error.ocurred_at).to be_within(1.second).of(Time.now)
```